controllers/usersignup/usersignup_controller_test.go

```diff
@@ -255,7 +279,7 @@ func TestUserSignupCreateSpaceAndSpaceBindingOk(t *testing.T) {
 				// then
 				require.Equal(t, reconcile.Result{}, res)
 				switch testname {
-				case "without skip space creation annotation", "with skip space creation annotation set to false", "with social event":
+				case "without skip space creation annotation", "with skip space creation annotation set to false", "with social event", "with feature toggles":
```

Comment:
should we check that the annotation is still there after the second reconcile ?

Reply:
Added here: https://github.com/codeready-toolchain/host-operator/pull/1048/commits/6951200098796af4e8846e7dd9603bdeb9ac2931